roles/fips/tasks/main.yml

```diff
@@ -0,0 +1,22 @@
+---
+- name: 'Install dracut-fips'
+  action: yum name=dracut-fips state=installed
```

Comment:
line breaks are added.

Reply:
I'm unfamiliar with action. Is there a reason plain ```yum``` doesn't work?

Reply:
Or can we use `package` module here? I also tend to prefer that we use syntax like:

```
package:
  name: dracut-fips
  state: present
```

Reply:
I just started learning Anisble and this is my first playbook. Thanks for commenting on that.
`Package` module makes more sense than `action` and `yum` module.